packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -92,11 +92,9 @@ class UnThemedLogRow extends PureComponent<Props, State> {
           </div>
         )}
         <div className={style.logsRowLevel} />
-        {
-          <div onClick={this.toggleDetails} className={style.logsRowToggleDetails}>
-            <i className={showDetails ? 'fa fa-chevron-up' : 'fa fa-chevron-down'} />
-          </div>
-        }
+        <div onClick={this.toggleDetails} className={style.logsRowToggleDetails}>
```

Comment:
Can we add a `title` here so it shows a tooltip to help with discovery of this feature?

Reply:
done